Formula/libevent.rb

```diff
@@ -15,7 +16,7 @@ class Libevent < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "openssl@1.1"
+  depends_on "libressl"
```

Comment:
I don't think this is the right approach. Much more software depends on openssl and mixing SSL libs usually ends badly.

Reply:
I agree